apps/aehttp/src/aehttp_dispatch_ext.erl

```diff
@@ -102,7 +104,8 @@ handle_request('PostBlock', Req, _Context) ->
 
 handle_request('PostTx', Req, _Context) ->
     SerializedTx = maps:get('SignedTx', Req),
-    SignedTx = aec_tx_sign:deserialize_from_binary(SerializedTx),
+    SignedTx = aec_tx_sign:deserialize_from_binary(
```

Comment:
is base64 encoding specific to HTTP protocol? Maybe we should lighten the controller and move it one of core modules?

Reply:
It's possible that the base64 encoding can be left out entirely. The JSON encoders/decoders should be able to handle a raw binary. But encoded JSON with escaped binaries is pretty hard on the eyes.

Strictly speaking it's not the network that justifies using base64, but rather more readable JSON.